### ISSUE TITLE
side="left" not working

### ISSUE THREAD

**Issue by frankthelen** [rpocklin/angular-timeline#21]
Hi!

I am using angular-timeline 1.6.1.
Putting side="left" on the <timeline-event> has no effect; the timeline is still rendered left-and-right (unless max-width 767px).

This is my markup:

``` html
<timeline>
    <timeline-event ng-repeat="event in events" side="left">
        <timeline-badge>...</timeline-badge>
        <timeline-panel>...</timeline-panel>
    </timeline-event>
</timeline>
```

Thanks,
Frank

**Comment by rpocklin** [comment]
It needs an angular variable - try putting it in quotes ie. `side="'left'"`

**Comment by frankthelen** [comment]
Hmmm, `side="'left'"` doesn't work either.

**Comment by rpocklin** [comment]
Can you make a plunkr?

**Comment by bortexz** [comment]
Same happens to me, side directive is not working.

**Comment by rpocklin** [comment]
You're right, a commit was missed, sorry guys.  Fixed in `v1.6.2`.
No need for quotes, use `side="left"` `side="right"` or `side="{{ exp }}"

Thanks for reporting it :)
